src/Contracts/IPriceable.php

```diff
@@ -27,13 +27,8 @@
  * This interface is in the services package to encourage making classes that describe items compatible across both the
  * seat core and plugin, even if they don't depend on recursivetree/seat-prices-core.
  */
-interface IPriceable extends HasTypeID
+interface IPriceable extends HasTypeID, HasTypeIDWithAmount
```

Comment:
```suggestion
interface IPriceable extends HasTypeIDWithAmount
```

since `HasTypeIDWithAmount` already extends `HasTypeID`, that means `IPriceable` extends `HasTypeID` when extending `HasTypeIDWithAmount`, isn't it ?